spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
@rmzelle normally you would do something like `expect { JSON.parse(x) }.to_not raise(JSON::ParserError)` but since you're actually parsed the file to populate the renamed style lists there is little point in parsing the file one more time.

I'd suggest to rename `JSON_IS_VALID` to something more descriptive though.

In general, we've been adding quite a number of global constants... this is not good style, but the reason why I started doing this was simply for performance to ensure every style is only parsed once. If you need the renamed style lists only in the repository test, however, I would just set it up using before hooks there.